app/services/UserService.py

```diff
@@ -31,3 +31,21 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users():
+    """
+    Retrieves all users.
+    """
+    users = await UserInDB.all()
+    return [PydanticUserResponse(**user.__dict__) for user in users]
```

Comment:
Please do not use the __dict__() method. Use the from_orm method. You should use the changes you suggested. It was not someting i asked you to remove, i simply explained what it does and how it should be used.
You can use orm_mode in this case.

Reply:
done